oximeter/oximeter/schema/database-transaction.toml

```diff
@@ -0,0 +1,26 @@
+format_version = 1
```

Comment:
Is the primary motivation for this shift "better detection/management of schema changes?"

Reply:
Yeah, that's the main reason. There are a bunch of pieces to this, tracked under #5941, and this is part of the "move all extant timeseries to TOML" item. Another good reason is that we can attach metadata, like the descriptions, that go in the DB but not each sample that `oximeter` collects. We can also generate docs for each timeseries, which is nice.